### ISSUE TITLE
Ob nether bug

### ISSUE THREAD

**Issue by KRISHN13** [MCIndiaNetwork/MCI-Server#37]
![20220122_101922](https://user-images.githubusercontent.com/95619189/150631935-5a723beb-db32-49b4-92e9-aa2288d47f5f.jpg)
Idk why when I go into my nether i spawn on these type of platform in place of the bedrock and I can't break that blocks idk why and also makes my nether look messy

**Comment by SpikeyHelmet** [comment]
Could you give us your in-game name so we can check this further?

**Comment by KRISHN13** [comment]
krishna1320006

**Comment by DestinedPhoenix** [comment]
Yes, this happens on my OB too. Try making a nether portal on some other place on your OB island. Just make sure that you don't press the movement keys as soon as you enter in, it then spawns you exactly on the edge.

**Comment by SpikeyHelmet** [comment]
will check

**Comment by KRISHN13** [comment]
Oke

**Comment by KRISHN13** [comment]
Uuuuh??
